docs-ref-services/keyvault.md

```diff
@@ -21,32 +21,54 @@ Safeguard and manage cryptographic keys and secrets used by cloud applications a
 
 ```XML
 <dependency>
-    <groupId>com.microsoft.azure</groupId>
-    <artifactId>azure-keyvault</artifactId>
-    <version>1.1.2</version>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-security-keyvault-secrets</artifactId>
+    <version>4.0.0</version>
+</dependency>
+
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-security-keyvault-keys</artifactId>
+    <version>4.0.0</version>
 </dependency>
 ```   
 
 ## Example
 
-Retrieve a [JSON web key](https://tools.ietf.org/html/draft-ietf-jose-json-web-key-18) from a Key Vault.
+
+Retrieve a [JSON web key](https://tools.ietf.org/html/draft-ietf-jose-json-web-key-18) and a secret from a Key Vault.
 
 ```java
-KeyVaultClient kvc = new KeyVaultClient(credentials);
-KeyBundle returnedKeyBundle = kvc.getKey(vaultUrl, keyName);
-JsonWebKey jsonKey = returnedKeyBundle.key();
+
+KeyClient keyClient = new KeyClientBuilder()
+        .vaultUrl(<your-vault-url>)
+        .credential(<your-credentials>)
+        .buildClient();
+KeyVaultKey key = keyClient.getKey("key_name");
+
+SecretClient secretClient = new SecretClientBuilder()
+		 .vaultUrl(<your-vault-url>)
+		 .credential(new DefaultAzureCredentialBuilder().build())
+		 .buildClient();
+KeyVaultSecret secret = secretClient.getSecret("secret_name");
+
 ```
+| Service | Package | Readme | Samples | API Reference | Changelog |
+| ------- | ------- | ------ | ------- | ------------- | --------- |
+| Key Vault - Keys | [azure-security-keyvault-keys - 4.0.0](https://search.maven.org/artifact/com.azure/azure-security-keyvault-keys/4.0.0/jar/) | [ReadMe](https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-keys_4.0.0/sdk/keyvault/azure-security-keyvault-keys/README.md) | [Samples](https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-keys_4.0.0/sdk/keyvault/azure-security-keyvault-keys/src/samples) | [Api Reference](https://azuresdkdocs.blob.core.windows.net/$web/java/azure-security-keyvault-keys/4.0.0/index.html) | [ChangeLog](https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-keys_4.0.0/sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md) |
+| Key Vault - Secrets | [azure-security-keyvault-secrets - 4.0.0](https://search.maven.org/artifact/com.azure/azure-security-keyvault-secrets/4.0.0/jar/) | [ReadMe](https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-secrets_4.0.0/sdk/keyvault/azure-security-keyvault-secrets/README.md) | [Samples](https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-secrets_4.0.0/sdk/keyvault/azure-security-keyvault-secrets/src/samples) | [Api Reference](https://azuresdkdocs.blob.core.windows.net/$web/java/azure-security-keyvault-secrets/4.0.0/index.html) | [ChangeLog](https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-secrets_4.0.0/sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md) |
 
-> [!div class="nextstepaction"]
```

Comment:
I'd keep this and as an underlying link I would point to the QS for secrets: [Get started with Secrets](/azure/key-vault/quick-create-java)

Reply:
done, please check if the link I added is correct

Reply:
works for me - would just change the "Explore the Client APIs" to "Get started with Secrets"

Reply:
updated, please check the path